Ccs811

gemu2015 opened the pull request:
support for CCS811 TVOC eCO2 sensor (much cheaper then SGP30)

with optional compensation for humidity and temperature

gemu2015 committed:
Merge remote-tracking branch 'arendst/development' into development

gemu2015 committed:
cfg

gemu2015 committed:
Merge remote-tracking branch 'arendst/development' into development

gemu2015 committed:
Merge remote-tracking branch 'arendst/development' into development

# Conflicts:
#	platformio.ini

gemu2015 committed:
pi

gemu2015 committed:
CCS811 added

gemu2015 committed:
size fix

gemu2015 committed:
temp res 0.25

gemu2015 committed:
HTU21 added SHT21

gemu2015 committed:
t/4

gemu2015 committed:
fixes

gemu2015 committed:
update from arends

gemu2015 committed:
update arends

gemu2015 committed:
update to 6.1.0a

gemu2015 committed:
Merge remote-tracking branch 'arendst/development' into CCS811

# Conflicts:
#	README.md

gemu2015 committed:
update to 6.1.1b

arendst commented:
I hope you do not expect me to verify 580 files before marging it?

Please re-PR with only the suggested file changes/updates.

gemu2015 committed:
fresh install from from master

arendst commented:
You're almost there ;-) Now I only have to check 605 files...

gemu2015 commented:
i only changed,added 4 files and added the library

xsns_31_cc811.ino => the CCS811 driver

since this chip can be compensated with temperature and RELATIVE humidity

i made 2 global vars in sonoff.inio

uint8_t glob_humidity=0;
sint16_t glob_temperature=-9999;


which are set currently only in xsns_14_sht3x.ino and xsns_09_bmp.ino because i own these 2 sensors


now i copied all files from the current version and made my changes again.

something broken in my git repo ?






> Am 23.07.2018 um 13:56 schrieb Theo Arends <notifications@github.com <mailto:notifications@github.com>>:
> 
> I hope you do not expect me to verify 580 files before marging it?
> 
> Please re-PR with only the suggested file changes/updates.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/arendst/Sonoff-Tasmota/pull/2558#issuecomment-407032425>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ALG4Y81TxtJdiXka7Jfro_n-VR0V6VjGks5uJbnWgaJpZM4Tqb23>.
>
